chromium/bootstrap.js

```diff
@@ -0,0 +1,7 @@
+function require(module) {
+  if (module.startsWith('./') && !module.hasOwnProperty(module.slice(2))) {
```

Comment:
What does `!module.hasOwnProperty(module.slice(2))` do?

Reply:
it strips off the leading `./` of the module name.  Since we do ` require('./background');` but the property on the object is called `background`. This is to be compatible with node's require, which uses relative paths.